src/panels/lovelace/cards/hui-error-card.ts

```diff
@@ -43,8 +43,8 @@ export class HuiErrorCard extends LitElement implements LovelaceCard {
     return css`
       :host {
         display: block;
-        background-color: #ef5350;
-        color: white;
+        background-color: var(--error-color);
+        color: var(--error-card-text-color, white);
```

Comment:
This is too specific, we want to use it everywhere we use the error color as the background.

Reply:
Where's that? I don't see anywhere else.

Reply:
Future proof :-)

Reply:
regardless, we should limit ourselves to be only on  a card

Reply:
So what's a good name? `error-text-color`? In some places `error-color` is used as the color of text.

Reply:
https://github.com/home-assistant/frontend/pull/6961#discussion_r487569117

Reply:
```suggestion
        color: var(--on-error-color, white);
```